src/sentry/integrations/bitbucket/integration.py

```diff
@@ -37,13 +38,57 @@ def get_client(self):
             self.model.external_id,
         )
 
+    def get_project_config(self):
+        client = self.get_client()
+        disabled = False
+        try:
+            repos = client.get_repos(self.model.name)
+        except ApiError:
+            # TODO(LB): Disable for now. Need to decide what to do with this in the future
+            # should a message be shown to the user?
+            # Should we try refreshing the token? For VSTS that often clears up the problem
+            repo_choices = []
+            disabled = True
+        else:
+            repo_choices = [(repo['uuid'], repo['full_name']) for repo in repos['values']]
+
+        try:
+            # TODO(LB): Will not work in the UI until the serliazer sends a `project_id` to get_installation()
+            # serializers and UI are being refactored and it's not worth trying to fix
+            # the old system. Revisit
+            default_repo = self.project_integration.config.get('default_repo')
+        except Exception:
+            default_repo = None
+
+        initial_repo = ('', '')
+        if default_repo is not None:
+            for repo_id, repo_name in repo_choices:
+                if default_repo == repo_id:
+                    initial_repo = (repo_id, repo_name)
+                    break
+
+        return [
+            {
+                'name': 'default_repo',
+                'type': 'choice',
+                'allowEmpty': True,
+                'disabled': disabled,
+                'required': True,
+                'choices': repo_choices,
+                'initial': initial_repo,
+                'label': _('Default Repository Name'),
+                'help': _('Enter the full name of the Bitbucket repo that you wish to use as the default for new issues'),
```

Comment:
Is this help wrong since it's a choice menu?

Reply:
Yup. Good catch